.github/workflows/main.yml

```diff
@@ -282,6 +282,7 @@ jobs:
         uses: ./.github/actions/setup-testing-rust
         with:
           rust-toolchain-version: ${{ env.RUST_TOOLCHAIN_VERSION }}
+          default: true
```

Comment:
this was previously deleted as `setup-testing-rust` has no such arg. I guess this was added by mistake during rebase?

Reply:
That's the last straw. I am going to look into upgrading my conflict-fixing experience 😅